cypress/integration/LocationEditor.spec.ts

```diff
@@ -4,7 +4,7 @@ const LOCATION_1 = {
 };
 const LOCATION_2 = {
   address: 'Max-Urich-Straße 1, 13355 Berlin, Germany',
-  value: { lon: 13.38372, lat: 52.53926 },
+  value: { lon: 13.3838105, lat: 52.5388562 },
```

Comment:
If it fails try these 👇🏻 (copied from master. Google Maps JS does some weird rounding sometimes)

```suggestion
  value: { lon: 13.38381, lat: 52.53885 },
```